pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -184,7 +184,7 @@ func (ev *Evaluator) Preempt(ctx context.Context, pod *v1.Pod, m framework.NodeT
 			},
 		}
 		// Specify nominatedNodeName to clear the pod's nominatedNodeName status, if applicable.
-		return framework.NewPostFilterResultWithNominatedNode(""), framework.NewStatus(framework.Unschedulable, fitError.Error())
+		return framework.NewPostFilterResultWithNominatedNode(""), framework.NewStatus(framework.UnschedulableAndUnresolvable, fitError.Error())
```

Comment:
This doesn't look right to me. The default preemption plugin is implemented in a **single-node** preemption manner, in other words, it hasn't iterated all preemption strategies. So we should keep `Unschedulable` to give other out-of-tree PostFilter plugins a chance to proceed with their custom logic (maybe cross-node preemption).

Reply:
+1; lets not change the current behavior of the existing preemption plugin.

Reply:
`We may have more preemption strategies` is right, revert the change to default preemption plugin.